### PR TITLE
Translate block labels

### PR DESCRIPTION
**Pull request by core23**
* [X] German
* [X] English
* [ ] French

### CODE REVIEW

**Comment by core23** [comment]
@rande @Soullivaneuh @sagikazarmark Can you help me with the translation?

File: Resources/translations/SonataMediaBundle.de.xliff [hunk 1]
```diff
@@ -453,6 +453,46 @@
...
+                <source>form.label_format</source>
+                <target>Format</target>
+            </trans-unit>
+            <trans-unit id="form.label_pause_time">
+                <source>form.label_pause_time</source>
+                <target>Pausenzeit</target>
+            </trans-unit>
+            <trans-unit id="form.label_start_paused">
+                <source>form.label_start_paused</source>
+                <target>Pausieren</target>
```
Comment (OskarStark) [inline Resources/translations/SonataMediaBundle.de.xliff]: ``Pausiert starten?``

or

``Mit Pause starten?``

File: Resources/translations/SonataMediaBundle.de.xliff [hunk 2]
```diff
@@ -453,6 +453,46 @@
...
+                <source>form.label_gallery</source>
+                <target>Galerie</target>
+            </trans-unit>
+            <trans-unit id="form.label_format">
+                <source>form.label_format</source>
+                <target>Format</target>
+            </trans-unit>
+            <trans-unit id="form.label_pause_time">
+                <source>form.label_pause_time</source>
+                <target>Pausenzeit</target>
```
Comment (OskarStark) [inline Resources/translations/SonataMediaBundle.de.xliff]: ``Pausenzeit zwischen den Bildern`` ??

File: Block/FeatureMediaBlockService.php [hunk 3]
```diff
@@ -47,19 +47,31 @@ public function buildEditForm(FormMapper $formMapper, BlockInterface $block)
...
+                    'label'    => 'form.label_title',
+                )),
+                array('content', 'textarea', array(
+                    'required' => false,
+                    'label'    => 'form.label_content',
+                )),
+                array('orientation', 'choice', array(
+                    'choices' => array(
+                        'left'  => 'form.feature_left_choice',
+                        'right' => 'form.feature_right_choice',
```
Comment (OskarStark) [inline Block/FeatureMediaBlockService.php]: should be ``form.label_orientation_left`` IMO, what do you think @core23

Comment (core23) [inline Block/FeatureMediaBlockService.php]: Thanks for your advice. Changed the label key

**Comment by OskarStark** [comment]
ping @rande can you please deliver the french version?

**Comment by sagikazarmark** [comment]
What is Pause time, start paused and wrap here?

**Comment by sagikazarmark** [comment]
(Also: have you though about using Crowdin?)

**Comment by core23** [comment]
> What is Pause time, start paused and wrap here?

Pause: the time between each image
Start paused: Should the block start in pause mode or should it play with autoration
wrap: No idea. Should be removed soon https://github.com/sonata-project/SonataMediaBundle/pull/890

What is crowdid?

**Comment by sagikazarmark** [comment]
Thanks.

Not sure pause is a good name here. It is usually called delay, isn't it?

https://crowdin.com/

**Comment by OskarStark** [comment]
delay sounds great, but we have to keep the field name ``pause``, do we?

crwodin is not free, or am i wrong?

**Comment by sagikazarmark** [comment]
They follow the good practice: free for open source:

https://crowdin.com/page/open-source-project-setup-request

Dunno if we have to keep it pause.

**Comment by OskarStark** [comment]
@sagikazarmark could you please open a separate issue for the crowdin stuff?

sounds interesting to me :+1:

**Comment by OskarStark** [comment]
Lets keep ``pause`` in this PR

**Comment by OskarStark** [comment]
Thank you @core23

**Pull request merged**